env.example

```diff
@@ -4,6 +4,16 @@ NODE_ENV=development
 # PORT is the port used by the web server
 PORT=3000
 
+
+# FRONTEND is the Front-End env variable that defines what Front-End
+# gets served for Telescope.
+# Currently, FRONTEND only has two supported values:
+#   'gatsby' - The Gatsby Front-End
+#   'next' - The NextJs Front-End
+# Our default value for FRONTEND is 'gatsby'
+FRONTEND='gatsby'
```

Comment:
Remove the surrounding quotes, all values are strings:

```
FRONTEND=gatsby
```

Reply:
Will do, thanks!